### PR TITLE
Helper methods to expose field mappings

### PR DESCRIPTION
**Pull request by rohkhann**
## Why make this change?
For customers consuming dab, it maybe useful to store field mappings internally and surface them to their end users. Currently these are private dictionaries and hence need to be regenerated if wanting to show on consumer side.

## What is this change?
Exposing the dictionaries publicly through readonlydictionary.

## How was this tested?
Existing unit tests should cover all scenarios.

### CODE REVIEW

**Comment by rohkhann** [comment]
azp/ run

**Review by severussundar — APPROVED** [review]

**Review by seantleonard — COMMENTED** [review]
Additions look good. However, pre-existing test suite does not cover these methods. Can you evaluate level of effort to add that coverage?

**Review by aaronburtle — APPROVED** [review]
Looks good to me. If you can add test coverage easily before merging, please do so.

File: src/Core/Services/MetadataProviders/SqlMetadataProvider.cs
[the remark this answers is not in the document]
Comment (rohkhann) [reply src/Core/Services/MetadataProviders/SqlMetadataProvider.cs:1]: Added tests!

**Pull request merged**